### PR TITLE
make sure all design types are supported

### PR DESCRIPTION
**Pull request by blongden73**
## What does this change?
Supports more card types in the story package container.

### CODE REVIEW

File: static/src/stylesheets/inline/story-package-garnett.scss [hunk 1]
```diff
@@ -34,27 +35,52 @@
...
         }
+
+        .fc-item__headline {
+            color: #ffffff;
+        }
     }
 
-    .fc-item__container.u-faux-block-link--hover {
-        background-color: darken($story-package-garnett, 5%);
+    .fc-item.fc-item--pillar-news.fc-item--type-comment,
```
Comment (sndrs) [inline static/src/stylesheets/inline/story-package-garnett.scss]: do you need `.fc-item`?

**Comment by blongden73** [comment]
@sndrs I removed that specificity...just wanted to be safe, but still works

File: static/src/stylesheets/inline/story-package-garnett.scss [hunk 2]
```diff
@@ -34,27 +35,52 @@
...
+
+            @include mq($from: tablet) {
+                top: 0px;
+            }
         }
     }
 
-    .fc-item {
+    .fc-item,
+    .fc-item.fc-item--pillar-opinion,
```
Comment (sndrs) [inline static/src/stylesheets/inline/story-package-garnett.scss]: same specificity here? is this needed?

File: static/src/stylesheets/inline/story-package-garnett.scss [hunk 3]
```diff
@@ -34,27 +35,52 @@
             background: -webkit-linear-gradient(top, #ffe500 0%, #ffe500 50%, #ffe500 100%);
             background: linear-gradient(to bottom,  #ffe500 0%, #ffe500 50%, #ffe500 100%);
```
Comment (sndrs) [inline static/src/stylesheets/inline/story-package-garnett.scss]: no need to add this prefix, will be auto-prefixed

File: static/src/stylesheets/inline/story-package-garnett.scss [hunk 4]
```diff
@@ -34,27 +35,52 @@
...
             display: inline;
-            box-shadow: 0px 0 0 2px #ffe500;
-            line-height: 1;
+            box-shadow: -4px 0 0 0 #ffe500;
+            line-height: 32px;
             top: 6px;
             position: relative;
+            padding-right: 4px;
```
Comment (sndrs) [inline static/src/stylesheets/inline/story-package-garnett.scss]: can you add a comment to explain why using magic numbers (if the need to be?)

**Review by AWare — APPROVED** [review]

**Pull request merged**